docs/proposals/federation.md

```diff
@@ -222,10 +222,14 @@ initial implementation targeting single cloud provider only.
 1. Auto-scaling (not yet available) in the remaining clusters takes
    care of it for me automagically as the additional failed-over
    traffic arrives (with some latency).
+1. I manually specify "additional resources to be provisioned" per
+   remaining cluster, possibly proportional to both the remaining functioning resources
+   and the unavailable resources in the failed cluster(s).
+   (All the benefits of over-provisioning, without expensive idle resources.)
```

Comment:
The drawback here is that you might not be able to acquire the resources.
If everyone is planning to do this, and a large IaaS provider/zone has a large outage, it isn't assured that everyone can instantly scale up on other IaaS providers/zones.